typescript/introduction/tsc-data-types-i/README.md

```diff
@@ -0,0 +1,9 @@
+name: Data Types I
+
+description: You might know these from JavaScript.
+
+insights:
+  - tsc-intro-data-types
+  - tsc-number-string-boolean
+  - tsc-array-tuple
+  - tsc-any-void-null
```

Comment:
this could go into 2nd workout if we decide to include `enum` in `tsc-array-tuple` 

moreover, I'd recommend we slipt them like this:
- intro data types
- number, string, boolean
- arrays
- enums
- tuples 

and 2nd workout:
- any-void-null
- never
- objects  

punctually, I think `enums` make more sense to be presented alongside arrays & tuples (as they all refer to pseudo-sequential types). similarily, `any-void-null` are special types that seem more similar to `never`

Reply:
Sure thing 👍